client/src/scenes/challenge/approval/index.jsx

```diff
@@ -99,8 +103,11 @@ class ChallengeApproval extends React.Component {
               key={id}
               id={id}
               {...rest}
-              onApprove={this.handleApprove}
-              onDisapprove={this.handleDisapproval}
+              hasEditableComments={false}
+              leftButtonText="Approve"
+              rightButtonText="Disapprove"
+              onLeftButtonClick={this.handleApprove(id)}
```

Comment:
What was the reason for this change?

Reply:
What part of the change?

Reply:
If you mean the `this.handleApprove(id)`, it is so that challenge can just generically pass a function without knowing "i need to pass an id to this" (note that `handleApprove(id)` now returns a function that takes no args). This way, the button click can just call the function passed and anything else that uses the challenge component can pass whatever they want